config/locales/models.fr.yml

```
@@ -162,6 +162,7 @@ fr:
         landing_topic_title: Titre des exemples
         landing_topics: Exemples
         message_under_landing_topics: Message sous les exemples
+        partner_url: Url du site partenaire
```

```suggestion
        partner_url: URL du site partenaire
```
majuscules pour les acronymes